src/session/session_api.c

```diff
@@ -1806,6 +1806,24 @@ __session_timestamp_transaction(WT_SESSION *wt_session, const char *config)
     API_END_RET(session, ret);
 }
 
+/*
+ * __session_timestamp_transaction_numeric --
+ *     WT_SESSION->timestamp_transaction_numeric method.
+ */
+static int
+__session_timestamp_transaction_numeric(WT_SESSION *wt_session, WT_TS_TXN_TYPE which, uint64_t ts)
+{
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session;
+
+    session = (WT_SESSION_IMPL *)wt_session;
+    SESSION_API_CALL_NOCONF(session, timestamp_transaction_numeric);
```

Comment:
Confused why this isn't `SESSION_API_CALL_PREPARE_ALLOWED(xxx)`, that is, why are the rules here different from `__session_timestamp_transaction()`?

Reply:
Because I didn't understand that we had to allow this after prepare - I'll fix it in light of the comments in the other thread on this.

Reply:
This turned out to be a surprisingly large PITA, it looks like it needs some form of autogenerated config entry in `config.h` - I could do with some advice here.

Reply:
There is already `SESSION_API_CALL_PREPARE_ALLOWED_NOCONF` in api.h, isn't that what you need?

Reply:
Yep - not sure how I managed to miss that, thanks for the pointer. Fixed in ca2c8c4